### ISSUE TITLE
Feature suggestions from another patched Whatsapp app/s

### ISSUE THREAD

**Issue by Turhvjbufv** [Schwartzblat/WhatsAppPatcher#51]
Currently I'm not using this app so some of these features might already be in the app. 
1. freeze last seen. 
2. View onetime images infinite times(I know we can currently save them but sometimes you don't to / you forget to save them).
3. Hide status view(don't tell someone that you viewed their status).
4. Show deleted status. 
5. Show blue tick(confirmation that you read the message)only after you replay. 
6. Make Whatsapp customizable (in other modified whatsapp apps you can customize almost any UI element that you want, they even have websites for downloading Whatsapp themes).
6. Disable audio notification. 
7. Increase forward limit.
8. Disable Whatsapp compressor(for images that you send).
9. Allow sending more images at once(instead of the current limit).
10. Add an 2 buttons to a status, one to download the status and one for showing the person that you viewed their status(depended on 3)
11. Hide contacts/groups(hide contacts and groups, but you can view them if you press on "WhatsApp" in header, which you can set a password for accessing that if you want(which means you would need to click on "Whatsapp" in header and enter the password to view the chats)).
12. Remove "read more..."(automatically open a message that would require you to press "read more..." To full length).
13. Group admin indicator( show some symbol next to the group admins name when they send a message, and show(the same symbol but with different color or  a different symbol entirely) if the admin is also the one who created the group).
14. Hide date and name when copying more than 2 messages. 
15. Allow enabling and disabling the proximity sensor.
I think these features would be nice to have. 
Thank you.

**Comment by roviansan** [comment]
(Freeze last seen) is a must imo..
